doc/resource/api/core.rst

```diff
@@ -1,79 +0,0 @@
-====================
- :mod:`core` Module
-====================
```

Comment:
I don't think this should have been nuked. I can't find a replacement elsewhere.